File: app/src/main/java/ua/com/amicablesoft/android/wr/dal/IVideoRepository.java
package ua.com.amicablesoft.android.wr.dal;

import java.util.List;

import ua.com.amicablesoft.android.wr.models.VideoFile;

/**
 * Created by olha on 2/27/17.
 */

interface IVideoRepository {

    List<VideoFile> getListVideoFiles(String powerlifter);
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/dal/IVideoRepository.kt
/*
 * Created by Olha Tymoshenko on 5/24/18 8:35 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.dal

import ua.com.amicablesoft.android.wr.models.VideoFile

interface IVideoRepository {

    fun getListVideoFiles(specification: Specification): List<VideoFile>?
}